docs/user/openstack/README.md

```diff
@@ -501,6 +501,14 @@ spec:
           availabilityZone: <optional_openstack_availability_zone>
 ```
 
+[provider-spec-definition]: https://github.com/openshift/cluster-api-provider-openstack/blob/155384b859c5b2fb5b7f11c9111d3f8e4f3066bd/pkg/apis/openstackproviderconfig/v1alpha1/types.go#L31
+
+#### Defining a MachineSet That Uses Multiple Networks
+
+To attach multiple networks to a MachineSet, the `primarySubnet` value in the `providerSpec` must be the same OpenStack subnet as the [machines subnet](./customization.md#cluster-scoped-properties) in the `install-config.yaml`. If this value is not set correctly, the nodes will be unable to join the cluster.
```

Comment:
Not set "correctly," or not set so that it contains `primarySubnet`? 

Just "correctly" could imply that there are a variety of ways to screw up the value, with what's in the previous sentence being just one of them. 

(Could expand if that's actually the case.)

Reply:
When defining a MachineSet with multiple networks, the `primarySubnet` value in the `providerSpec` must be set to the OpenStack subnet that you want the Kubernetes endpoints of nodes to be published on. For most use cases, this is the same subnet as the [machinesSubnet](./customization.md#cluster-scoped-properties) in the `install-config.yaml`.

How is this? I think it more accurately reflects the usage

Reply:
Works for me. :+1:

Reply:
Take a look at what I pushed and see if it reads more clearly to you